docs/pages/get-started/faq.md

```diff
@@ -360,9 +385,11 @@ Ephemera is researching various consensus protocols that would allow the network
 
 ### Will I be able to run my own XMTP node?
 
-Yes, you will be able to run a node.
+Yes, you will be able to run a node. 
 
-Ephemera is working toward a phased decentralization of the network.
+Ephemera is in the process of building the replication mechanisms required to enable third-parties to run XMTP nodes and participate in network decentralization.
+
+To follow and provide feedback on this work, see the [Replication tracking task](https://github.com/xmtp/xmtpd/issues/118) in the `xmtpd` repo.
```

Comment:
This is a bit murky depending on the definition of 'node' - the first version of decentralization involves a permissioned set of high-quality originator nodes from third-party organizations decided by a committee. So it's not the case that it's as easy as running a node out of your basement, there are a few additional hoops to jump through. It may be the case though that running a 'gateway node' is frictionless, but we haven't ironed out all of the details there.

I wonder if we should remove the definitive statement `Yes, you will be able to run a node`, which is technically correct but could give the wrong impression.

Reply:
Thank you for this, @richardhuaaa - again, what thoughtful feedback. I have removed the definitive statement as suggested. And yes, let's add a way for potential third-party organizations and individuals to express interest! Let's add it once we have a clearer picture of how that will work. <3